### PR TITLE
heading height fixed.

### LINKED ISSUE #6: heading overlapped with paragraph
Screenshot (1289)

Institute management portal heading inside a card overlapped with text when screen size changed.

i would like to work on this issue. also
@iiitv/project-maintainers please do let me know about this issue validity.

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes: #6  
you can see the updated site here : https://cranky-mclean-5f049c.netlify.app/

Screenshot (1300)

### CODE REVIEW

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir please review this pull request.

File: style.css [hunk 1]
```diff
@@ -267,6 +267,9 @@ nav {
   height: 2.8em;
   margin: 0px;
 }
+ .heading-height{
```
Comment (aashutoshrathi) [inline style.css]: Fix indentation here

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Also, activate your Windows 😛

**Comment by nlok5923** [comment]
@aashutoshrathi Sir requested changes done.

**Review by aashutoshrathi — APPROVED** [review]
LGTM

**Pull request merged**